python/cudf/cudf/tests/test_orc.py

```diff
@@ -1379,3 +1379,14 @@ def test_names_in_struct_dtype_nesting(datadir):
     edf = cudf.DataFrame(expect.to_pandas())
     # test schema
     assert edf.dtypes.equals(got.dtypes)
+
+
+def test_writer_lists_structs():
+    df_in = cudf.read_orc(list_struct_buff)
```

Comment:
Is there a reason that `list_struct_buff` is a module-scope variable rather than a fixture? If it's only ever read, could it be a module-scope fixture?